modules/cuml/src/metrics.cpp

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2020-2021, NVIDIA CORPORATION.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <node_cuda/utilities/error.hpp>
+#include <node_cudf/utilities/napi_to_cpp.hpp>
+#include <node_cuml/metrics.hpp>
+#include <node_rmm/device_buffer.hpp>
+
+#include <metrics/trustworthiness_c.h>
```

Comment:
Do we need to include a private cuML header here? This seems like it should be in `<cuml/metrics/metrics.hpp>`? Maybe we need to send them a PR to promote it to a public API?

```suggestion
```

Reply:
Made a PR to do this here: https://github.com/rapidsai/cuml/pull/3968